Tooltip: Add Getting Started (#7831)
DevExpress/devextreme-documentation: concepts/05 UI Components/Tooltip/02 Getting Started with Tooltip/10 Customize Tooltip.md

                    <div class="blue-tooltip">Info</div>
                </template>
            </DxTooltip>
        </div>
    </template>

##### React

To customize Tooltip content, specify [contentRender](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentRender) or [contentComponent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentComponent):

    <!-- tab: App.tsx -->
    function likeRender(): JSX.Element {
        return <div>Like</div>;
    }

    function trashRender(): JSX.Element {
        return (
            <div className='red-tooltip'>Delete</div>
        );
    }

    function infoRender(): JSX.Element {
        return (
            <div className='blue-tooltip'>Info</div>
        );
    }

    function App(): JSX.Element {
        return (
            <div className="tooltips">
                <Tooltip
                    contentRender={likeRender}
                />
                <Tooltip
                    contentRender={trashRender}
                />
                <Tooltip
                    contentRender={infoRender}
                />
            </div>
        )
    }

---

> You can customize the appearance of Tooltip visual elements with CSS. This example implements the following styles to change the colors of `#trash-tooltip` and `#info-tooltip`:

Reviewer: As I see, not all platforms use these IDs - "trash-tooltip" and "info-tooltip". For example, React sample doesn't have those names. Maybe we should just mention associated classes. And maybe we need to mention that we specifies CSS class names in code snippets above. 

```suggestion
You can use CSS to customize Tooltip appearance. You may have noticed that code snippets above assigned CSS classes to all tooltips. This example defines those styles (`.red-tooltip` and `.blue-tooltip`):
```

> You can use CSS to customize Tooltip appearance. You may have noticed that code snippets above assigned CSS classes to all tooltips. This example defines those styles (`.red-tooltip` and `.blue-tooltip`):

Reviewer 2: ```suggestion
You can use CSS to customize the Tooltip's appearance. The code snippets above assigned CSS classes to all tooltips. This example defines those styles (`.red-tooltip` and `.blue-tooltip`):
```
